diff --git a/wire/extracted_peer_wire_csv b/wire/extracted_peer_wire_csv
index 5a2a8c23f..7b26242e3 100644
--- a/wire/extracted_peer_wire_csv
+++ b/wire/extracted_peer_wire_csv
@@ -95,6 +95,9 @@ msgdata,update_add_htlc,amount_msat,u64,
 msgdata,update_add_htlc,payment_hash,sha256,
 msgdata,update_add_htlc,cltv_expiry,u32,
 msgdata,update_add_htlc,onion_routing_packet,byte,1366
+msgdata,update_add_htlc,tlvs,update_add_tlvs,
+tlvtype,update_add_tlvs,blinding,2
+tlvdata,update_add_tlvs,blinding,blinding,point,
 msgtype,update_fulfill_htlc,130
 msgdata,update_fulfill_htlc,channel_id,channel_id,
 msgdata,update_fulfill_htlc,id,u64,
