--- wire/peer_wire.csv	2022-07-18 13:49:29.079542016 +0530
+++ -	2022-07-18 13:58:17.706696582 +0530
@@ -249,7 +249,11 @@
 msgdata,pong,byteslen,u16,
 msgdata,pong,ignored,byte,byteslen
 msgtype,peer_storage,7
+msgdata,peer_storage,len,u16,
+msgdata,peer_storage,blob,byte,len
 msgtype,peer_storage_retrieval,9
+msgdata,peer_storage_retrieval,len,u16,
+msgdata,peer_storage_retrieval,blob,byte,len
 tlvtype,n1,tlv1,1
 tlvdata,n1,tlv1,amount_msat,tu64,
 tlvtype,n1,tlv2,2
