--- wire/peer_wire.csv	2023-02-02 17:51:50.435463786 -0600
+++ -	2023-02-02 17:51:56.693837258 -0600
@@ -62,13 +63,13 @@
 msgdata,tx_signatures,channel_id,channel_id,
 msgdata,tx_signatures,txid,sha256,
 msgdata,tx_signatures,num_witnesses,u16,
-msgdata,tx_signatures,witness_stack,witness_stack,num_witnesses
+msgdata,tx_signatures,witnesses,witness_stack,num_witnesses
 subtype,witness_stack
-subtypedata,witness_stack,num_input_witness,u16,
-subtypedata,witness_stack,witness_element,witness_element,num_input_witness
+subtypedata,witness_stack,num_witness_elements,u16,
+subtypedata,witness_stack,witness_elements,witness_element,num_witness_elements
 subtype,witness_element
 subtypedata,witness_element,len,u16,
-subtypedata,witness_element,witness,byte,len
+subtypedata,witness_element,witness_data,byte,len
 msgtype,tx_init_rbf,72
 msgdata,tx_init_rbf,channel_id,channel_id,
 msgdata,tx_init_rbf,locktime,u32,
@@ -145,7 +146,7 @@
 tlvdata,channel_ready_tlvs,short_channel_id,alias,short_channel_id,
 msgtype,open_channel2,64
 msgdata,open_channel2,chain_hash,chain_hash,
-msgdata,open_channel2,zerod_channel_id,channel_id,
+msgdata,open_channel2,temporary_channel_id,channel_id,
 msgdata,open_channel2,funding_feerate_perkw,u32,
 msgdata,open_channel2,commitment_feerate_perkw,u32,
 msgdata,open_channel2,funding_satoshis,u64,
@@ -161,6 +162,7 @@
 msgdata,open_channel2,delayed_payment_basepoint,point,
 msgdata,open_channel2,htlc_basepoint,point,
 msgdata,open_channel2,first_per_commitment_point,point,
+msgdata,open_channel2,second_per_commitment_point,point,
 msgdata,open_channel2,channel_flags,byte,
 msgdata,open_channel2,tlvs,opening_tlvs,
 tlvtype,opening_tlvs,upfront_shutdown_script,0
@@ -173,7 +175,7 @@
 tlvtype,opening_tlvs,require_confirmed_inputs,2
 tlvdata,opening_tlvs,require_confirmed_inputs,empty,byte,0
 msgtype,accept_channel2,65
-msgdata,accept_channel2,zerod_channel_id,channel_id,
+msgdata,accept_channel2,temporary_channel_id,channel_id,
 msgdata,accept_channel2,funding_satoshis,u64,
 msgdata,accept_channel2,dust_limit_satoshis,u64,
 msgdata,accept_channel2,max_htlc_value_in_flight_msat,u64,
@@ -187,6 +186,7 @@
 msgdata,accept_channel2,delayed_payment_basepoint,point,
 msgdata,accept_channel2,htlc_basepoint,point,
 msgdata,accept_channel2,first_per_commitment_point,point,
+msgdata,accept_channel2,second_per_commitment_point,point,
 msgdata,accept_channel2,tlvs,accept_tlvs,
 tlvtype,accept_tlvs,upfront_shutdown_script,0
 tlvdata,accept_tlvs,upfront_shutdown_script,shutdown_scriptpubkey,byte,...
