--- wire/peer_wire.csv	2023-01-09 12:09:54.439255190 -0600
+++ -	2023-01-09 12:15:37.608035051 -0600
@@ -171,6 +173,7 @@
 tlvtype,opening_tlvs,request_funds,3
 tlvdata,opening_tlvs,request_funds,requested_sats,u64,
 tlvdata,opening_tlvs,request_funds,blockheight,u32,
+tlvtype,opening_tlvs,require_confirmed_inputs,2
 msgtype,accept_channel2,65
 msgdata,accept_channel2,zerod_channel_id,channel_id,
 msgdata,accept_channel2,funding_satoshis,u64,
@@ -190,7 +191,8 @@
 tlvdata,accept_tlvs,upfront_shutdown_script,shutdown_scriptpubkey,byte,...
 tlvtype,accept_tlvs,channel_type,1
 tlvdata,accept_tlvs,channel_type,type,byte,...
+tlvtype,accept_tlvs,require_confirmed_inputs,2
-tlvtype,accept_tlvs,will_fund,2
+tlvtype,accept_tlvs,will_fund,3
 tlvdata,accept_tlvs,will_fund,signature,signature,
 tlvdata,accept_tlvs,will_fund,lease_rates,lease_rates,
 subtype,lease_rates
