--- wire/peer_exp_wire.csv	2022-06-22 19:07:24.000000000 -0500
+++ -	2022-06-30 16:00:51.000000000 -0500
@@ -65,9 +57,12 @@
 msgdata,tx_signatures,txid,sha256,
 msgdata,tx_signatures,num_witnesses,u16,
 msgdata,tx_signatures,witnesses,witness,num_witnesses
+msgdata,tx_signatures,tlvs,txsigs_tlvs,
 subtype,witness
 subtypedata,witness,len,u16,
 subtypedata,witness,witness_data,byte,len
+tlvtype,txsigs_tlvs,shared_input_signature,0
+tlvdata,txsigs_tlvs,shared_input_signature,signature,signature,
 msgtype,tx_init_rbf,72
 msgdata,tx_init_rbf,channel_id,channel_id,
 msgdata,tx_init_rbf,locktime,u32,
