--- wire/peer_exp_wire.csv	2021-05-17 09:30:02.302260471 +0930
+++ -	2021-05-31 12:20:36.390910632 +0930
@@ -120,6 +82,9 @@
 subtypedata,lease_rates,channel_fee_max_proportional_thousandths,u16,
 subtypedata,lease_rates,lease_fee_base_sat,u32,
 subtypedata,lease_rates,channel_fee_max_base_msat,tu32,
+msgtype,stfu,2
+msgdata,stfu,channel_id,channel_id,
+msgdata,stfu,initiator,u8,
 msgtype,shutdown,38
 msgdata,shutdown,channel_id,channel_id,
 msgdata,shutdown,len,u16,
