--- wire/peer_exp_wire.csv	2020-09-16 20:59:05.170975876 -0500
+++ -	2020-10-22 16:56:33.413063210 -0500
@@ -33,7 +33,7 @@
 tlvdata,n2,tlv2,cltv_expiry,tu32,
 msgtype,tx_add_input,66
 msgdata,tx_add_input,channel_id,channel_id,
-msgdata,tx_add_input,serial_id,u16,
+msgdata,tx_add_input,serial_id,u64,
 msgdata,tx_add_input,prevtx_len,u16,
 msgdata,tx_add_input,prevtx,byte,prevtx_len
 msgdata,tx_add_input,prevtx_vout,u32,
@@ -48,16 +48,16 @@
 msgdata,tx_add_input,script,byte,redeemscript_len
 msgtype,tx_add_output,67
 msgdata,tx_add_output,channel_id,channel_id,
-msgdata,tx_add_output,serial_id,u16,
+msgdata,tx_add_output,serial_id,u64,
 msgdata,tx_add_output,sats,u64,
 msgdata,tx_add_output,scriptlen,u16,
 msgdata,tx_add_output,script,byte,scriptlen
 msgtype,tx_remove_input,68
 msgdata,tx_remove_input,channel_id,channel_id,
-msgdata,tx_remove_input,serial_id,u16,
+msgdata,tx_remove_input,serial_id,u64,
 msgtype,tx_remove_output,69
 msgdata,tx_remove_output,channel_id,channel_id,
-msgdata,tx_remove_output,serial_id,u16,
+msgdata,tx_remove_output,serial_id,u64,
 msgtype,tx_complete,70
 msgdata,tx_complete,channel_id,channel_id,
 msgtype,tx_signatures,71
@@ -125,7 +125,9 @@
 msgdata,funding_locked,next_per_commitment_point,point,
 msgtype,open_channel2,64
 msgdata,open_channel2,chain_hash,chain_hash,
-msgdata,open_channel2,feerate_per_kw_funding,u32,
+msgdata,open_channel2,feerate_funding_max,u32,
+msgdata,open_channel2,feerate_funding_min,u32,
+msgdata,open_channel2,feerate_funding_best,u32,
 msgdata,open_channel2,funding_satoshis,u64,
 msgdata,open_channel2,dust_limit_satoshis,u64,
 msgdata,open_channel2,max_htlc_value_in_flight_msat,u64,
@@ -148,6 +150,7 @@
 msgtype,accept_channel2,65
 msgdata,accept_channel2,channel_id,channel_id,
 msgdata,accept_channel2,funding_satoshis,u64,
+msgdata,accept_channel2,feerate_funding,u32,
 msgdata,accept_channel2,dust_limit_satoshis,u64,
 msgdata,accept_channel2,max_htlc_value_in_flight_msat,u64,
 msgdata,accept_channel2,htlc_minimum_msat,u64,
