--- onion_wire.csv	2021-08-25 12:41:02.872253965 +0930
+++ onion_wire.csv.raw	2021-08-25 13:52:00.748767887 +0930
@@ -8,6 +8,10 @@ tlvdata,tlv_payload,short_channel_id,short_channel_id,short_channel_id,
 tlvtype,tlv_payload,payment_data,8
 tlvdata,tlv_payload,payment_data,payment_secret,byte,32
 tlvdata,tlv_payload,payment_data,total_msat,tu64,
+tlvtype,onionmsg_payload,reply_path,2
+tlvdata,onionmsg_payload,reply_path,first_node_id,point,
+tlvdata,onionmsg_payload,reply_path,blinding,point,
+tlvdata,onionmsg_payload,reply_path,path,onionmsg_path,...
 tlvtype,onionmsg_payload,obs_next_node_id,4
 tlvdata,onionmsg_payload,obs_next_node_id,node_id,point,
 tlvtype,onionmsg_payload,obs_next_short_channel_id,6
@@ -29,10 +29,16 @@ tlvtype,onionmsg_payload,invoice,66
 tlvdata,onionmsg_payload,invoice,invoice,byte,...
 tlvtype,onionmsg_payload,invoice_error,68
 tlvdata,onionmsg_payload,invoice_error,invoice_error,byte,...
+tlvtype,encmsg_tlvs,padding,1
+tlvdata,encmsg_tlvs,padding,pad,byte,...
 tlvtype,encmsg_tlvs,next_node_id,4
 tlvdata,encmsg_tlvs,next_node_id,node_id,point,
 tlvtype,encmsg_tlvs,obs_next_short_channel_id,6
 tlvdata,encmsg_tlvs,obs_next_short_channel_id,short_channel_id,short_channel_id,
+tlvtype,encmsg_tlvs,next_blinding,12
+tlvdata,encmsg_tlvs,next_blinding,blinding,point,
+tlvtype,encmsg_tlvs,self_id,14
+tlvdata,encmsg_tlvs,self_id,data,byte,...
 subtype,onionmsg_path
 subtypedata,onionmsg_path,node_id,point,
 subtypedata,onionmsg_path,enclen,u16,
